src/main/resources/edu/cornell/kfs/concur/cu-spring-concur.xml

```diff
@@ -30,6 +30,7 @@
 		<property name="jobNames">
 			<list>
 				<value>concurEventNotificationProcessingBatchJob</value>
+				<value>concurRequestExtractCreatePdpFeedJob</value>
```

Comment:
Would it make more sense to call the new batch job: concurRequestExtractToPdpFeedBatchJob?